common/dtd/ldml.dtd

```diff
@@ -2825,10 +2826,11 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 
 <!ELEMENT cr ( #PCDATA ) >
 <!ATTLIST cr alt NMTOKENS #IMPLIED >
-    <!--@MATCH:literal/variant-->
+    <!--@MATCH:literal/variant, proposed-->
```

Comment:
ouch… that's not recent

Reply:
Yes, the changes in the test exposed some gaps.